### ISSUE TITLE
[Feature Request] Allow transfer of org owner

### ISSUE THREAD

**Issue by golanglemonade** [theopenlane/core#69]
Implement resolver to transfer ownership of organization 
     - if done by the current owner, request should go through if the new user exists in the org
     - current user should be updated from owner -> admin
     - new owner should be updated as owner
     - any related tuples in FGA should be added/deleted as required
     - If the user is not in the organization, they should be sent an invitation to the organization, once accepted the transfer of ownership should happen
     - if done by another user in the org that is not the owner, request should be denied

**Comment by golanglemonade** [comment]
#2 resolved in https://github.com/theopenlane/core/pull/446

**Comment by manab-pr** [comment]
@golanglemonade  is it still not resolved ? **assign** it to me if it's not.

**Comment by golanglemonade** [comment]
Nope, not resolved. I updated it earlier today to remove the parts that were originally resolved. Thanks @manab-pr!
